.buildkite/pipeline.nix

```diff
@@ -0,0 +1,172 @@
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+let
+  native_packaging_changes_regexes = [
+    "docker\/package/\.*"
+    "docker\/docker-tezos-packages\.sh"
+    "meta\.json"
+    "protocols\.json"
+    "nix\/nix\/sources\.json"
+  ];
+  static_binaries_changes_regexes = [
+    "docker\/build/\.*"
+    "docker\/docker-static-build\.sh"
+    "nix\/nix\/sources\.json"
+    "meta\.json"
+    "protocols\.json"
+  ];
+  nix_changes_regexes = [
+    ".*\.nix"
+    "protocols\.json"
+  ];
```

Comment:
It doesn't seem enough for nix jobs, e.g. there is `sources.json` file and some `.patch` files. But maybe for nix it's simpler to not use any conditions, because it knows itself what to rebuild and what not to rebuild.